packages/v4/src/content/design-guidelines/components/calendar-month/calendar-month.md

```diff
@@ -0,0 +1,17 @@
+---
+id: Calendar month
+section: components
+related: ["Date picker", "Time picker"]
+---
+
+A **calendar month** component allows users to select and navigate between days/ months/ years. This component is usually used with the date picker component, to display date options and selections. For more information about usage, view also [date picker guidelines](/components/date-picker/design-guidelines).
```

Comment:
Can you change "day/ months/ years" so that it reads "to select and navigate between **days, months and/or years**."

Also in the last sentences, can you get rid of the word "also" and just say "For more information about usage, view date picker guidelines"